.github/workflows/i18n-consistency-checker.yaml

```diff
@@ -17,88 +12,70 @@ jobs:
       matrix:
         language: [ja, zh]
     steps:
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
         with:
           fetch-depth: '0'
-      - name: Check consistency
+      - name: Check consistency and create issue
         id: check-consistency
+        env:
+          GH_TOKEN: ${{ github.token }}
         run: |
-          # Set the issue header
-          issue="\
-          # i18n Contents Consistency Issue\\n\
-          \\n\
-          The following files may have consistency issues with the English version. Please check and update the files.\\n\
-          \\n\
-          This issue is created when there is an update to content/en. It compares the git update history to let you know what updates are overdue. The issue should be closed when the update is complete.\\n"
-          
-          for file in $(find patterns/2-structured -name '*.md'); do
-          
-            # Get the translated file name and check if it exists
-            i18n_filename=$(echo "$file" | sed 's/patterns\/2-structured/translation\/${{matrix.language}}\/patterns/g')
+          # Declare the flags
+          declare -A flags=( ["ja"]=":jp: Japanese" ["zh"]=":cn: Chinese")
```

Comment:
Where to lookup the icons to use here?

Reply:
@yuhattor this is the only question remaining that I think we should add to the documentation. 

If you could share where to look up the strings like `:jp:`, then I can add that to the documentation.
I might do when working on #582.